app/src/test/kotlin/social/plasma/RelayTest.kt

```diff
@@ -0,0 +1,20 @@
+package social.plasma
+
+import social.plasma.relay.Relays
+import kotlinx.coroutines.flow.take
+import kotlinx.coroutines.flow.toList
+import kotlinx.coroutines.runBlocking
+import okhttp3.OkHttpClient
+import org.junit.Test
+
+class RelayTest {
+
+    @Test
+    fun openWebSocket() = runBlocking {
```

Comment:
test works as long as damus.io is up and busy.
should have a fake relay for this to be a true unit test.